hw/ip/rv_core_ibex/doc/boot-rom-patching.md

```diff
@@ -175,51 +175,67 @@ In this case, the dedicated SRAM can be locked after being loaded so that the pa
 
 A patch is a short sequence of instructions that replace some from the original code.
 The replacement is performed by re-routing instruction fetches to fetch from alternative addresses.
-This re-routing is handled by a block that sits between the Ibex core and TL-UL crossbar.
-The block is configured by 32 *patch match registers*.
+This re-routing is handled by a logic block that sits between the Ibex core and TL-UL crossbar.
+The block is configured by 32 sets of 4 match registers:
```

Comment:
I think we might also want to drop "match" here. Before, we were declaring a new name ("patch match register"), which we defined. With the new text, I think we're using "match register", but I don't think there's a definition. (And I don't think there needs to be!)